src/dtclient/kubernetes_settings.go

```diff
@@ -67,33 +80,50 @@ type constraintViolations []struct {
 	Path              string
 }
 
-func (dtc *dynatraceClient) CreateOrUpdateKubernetesSetting(clusterLabel, kubeSystemUUID, scope string) (string, error) {
-	if kubeSystemUUID == "" {
-		return "", errors.New("no kube-system namespace UUID given")
+const (
+	schemaId                                    = "builtin:cloud.kubernetes"
+	defaultSchemaVersion                        = "1.0.27"
+	hierarchicalMonitoringSettingsSchemaVersion = "3.0.0"
+)
+
+func (dtc *dynatraceClient) GetSchemasVersion(schemaId string) (string, error) {
```

Comment:
I don't know if things changed, but didn't we want to skip the schema version check and just attempt to first post schema v3 and if it fails fall back to v1?

this would mean we have the same number of requests if the v3 schema is missing and only a single request if it's present/available.

Reply:
is there a nice a reliable way to notice that its a fail because the schema v3 is not present, but otherwise everything is OK? 
I would prefer to avoid handling weird Status codes, and also always swallowing the error for the creation using v3 is not a nice thing aswell.

Reply:
I have change code according request of Christian and description in original issue, although to my opinion previous solution with version check was better. Please let me know in case we need to change it back.